### PR TITLE
draft: Admin app

### PR DESCRIPTION
**Pull request by heyMP**
Adding `/admin` route in the frontend.

### CODE REVIEW

**Comment by heyMP** [comment]
Hey @bennypowers, currently running into an issue importing our 2.0 elements into our frontend app.  I think it's related to this [issue](https://github.com/microsoft/TypeScript/issues/38538) but none of the fixes I've found are doing the trick.  

```
➜  mpotter-mac as michael  2022-game-app (admin-app) ✔
   yarn workspace frontend run dev
Web Dev Server started...

  Root dir: /Users/michael/Projects/2022-game-app/frontend
  Local:    http://localhost:8004/
  Network:  http://192.168.0.67:8004/

Error while handling server request.
[Error: ENOENT: no such file or directory, stat '/Users/michael/Projects/2022-game-app/node_modules/@patternfly/pfe-core/decorators.js.js'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'stat',
  path: '/Users/michael/Projects/2022-game-app/node_modules/@patternfly/pfe-core/decorators.js.js',
  expose: true,
  statusCode: 404,
  status: 404
}
```

**Comment by bennypowers** [comment]
looks like a rollup thing maybe

**Comment by heyMP** [comment]
@bennypowers am I in the ballpark?  I've tried to 'exclude' or specify the node_modules directory in a few different places, tsconfig, esbuild, rollup/web-dev to no avail.

Screen Shot 2022-03-14 at 4 25 26 PM

**Comment by bennypowers** [comment]
try that

**Comment by heyMP** [comment]
Hmmm... didn't seem to unfortunately.

**Comment by bennypowers** [comment]
I believe this is PFE's problem. we need to explicitly specify every importable file in our export map. I will work to update the pfe packages

**Comment by bennypowers** [comment]
copy the package.json from the abovelinked pr into your local node_modules and try again, it should work.

File: .yarn/patches/@patternfly-pfe-core-npm-2.0.0-next.1-e0ea84a032 [hunk 1]
```diff
@@ -0,0 +1,113 @@
+diff --git a/package.json b/package.json
+index 50076e841533b3154e6063de906d86ed823da705..604728d1d5f666f92da7c15993dd50cdecf6d31e 100644
+--- a/package.json
```
Comment (bennypowers) [inline .yarn/patches/@patternfly-pfe-core-npm-2.0.0-next.1-e0ea84a032:3]: https://www.npmjs.com/package/@patternfly/pfe-core

`@patternfly/pfe-core@2.0.0-next.2` should fix this

**Pull request merged**